service/src/main/java/com/codeforcommunity/processor/AuthDatabase.java

```diff
@@ -60,4 +66,37 @@ public boolean isOnBlackList(String signature) {
 
         return count >= 1;
     }
+
+    public void createSecretKey(int userId, String token) throws AuthException {
+        if (!doesUserExist(userId)) {
+            throw new AuthException("User does not exist.");
+        }
+
+        db.insertInto(Tables.VERIFICATION_KEYS).columns(VERIFICATION_KEYS.ID, VERIFICATION_KEYS.USER_ID)
```

Comment:
Use jOOQ records for inserting, replace this syntax

Reply:
Done.